whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
 %-
 % ====================================================================
 
+\newcommand{\red}[1]{\textcolor{red}{#1}}
+
 \section{Weak Lensing}
 \def\secname{wl}\label{sec:\secname}
 
-\credit{StephenRidgway},
 \credit{jmeyers314},
-\credit{tonytyson}.
+\credit{tonytyson},
+\credit{StephenRidgway}.
 
-Much of LSST cosmology may be limited by systematic errors rather than
-photon signal-to-noise. This is especially true of weak gravitational
-lensing,  which relies on very accurate (\ie low bias), but very low
-signal-to-noise, measurements of the shapes of galaxies, and high
-signal-to-noise measurements of PSF calibration stars. As outlined in the SRD,
-uniformity of seeing in the bands used for WL and special observing strategies 
-are required in order to reduce additive and multiplicative shear systematics.
+Much of LSST cosmology may be limited by systematic rather than statistical
+errors.  This is especially true of weak gravitational lensing, which relies on
+very accurate (\ie low bias), but very noisy, measurements of the shapes of many
```

Comment:
Yes, that's what we meant.

On 6/29/16 6:42 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69057792>:
>
>>
>> -Much of LSST cosmology may be limited by systematic errors rather than
>> -photon signal-to-noise. This is especially true of weak gravitational
>> -lensing,  which relies on very accurate (\ie low bias), but very low
>> -signal-to-noise, measurements of the shapes of galaxies, and high
>> -signal-to-noise measurements of PSF calibration stars. As outlined in the SRD,
>> -uniformity of seeing in the bands used for WL and special observing strategies
>> -are required in order to reduce additive and multiplicative shear systematics.
>> +Much of LSST cosmology may be limited by systematic rather than statistical
>> +errors.  This is especially true of weak gravitational lensing, which relies on
>> +very accurate (\ie low bias), but very noisy, measurements of the shapes of many
>
> We need very accurate ensemble shear estimates, not very accurate shape
> measurements of individual galaxies.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69057792>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6YvYw9qkdd_kjfyQgL9YsFHLxumLuks5qQx8egaJpZM4I9hd->.
>